src/workspace.scss

```diff
@@ -154,6 +156,9 @@
   .pf-l-stack__secondary::before {
```

Comment:
sorry I missed this on the other PR. I think this is an unnecessary overhead. We should not do it here.

Reply:
If we remove the dummy content and we don't include any text in the property, we won't have anyway to annotate what they are looking at.

Reply:
For clarity I think we should do one or the other.

Reply:
I'll remove this and add the text back to the components.

Reply:
To keep this PR in scope, I'll make another PR though.

Reply:
https://github.com/patternfly/patternfly-next/pull/82

Reply:
https://github.com/patternfly/patternfly-next/pull/82